src/plugins/ansible/README.md

```diff
@@ -0,0 +1,71 @@
+- infos = Information about the ansible plugin is in keys below
+- infos/author = Maximilian Irlinger <max@maxirlinger.at>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage/ansible
```

Comment:
Having `storage` here will cause the import and export shell tests to include this plugin. Which is not good, because we can not import using this plugin.

Do we have a better option or should I just leave `info/provides` blank?

Reply:
In #666 #3504 we tried to improve the infos/status by introducing infos/features/storage to provide such information. Probably we should add an import/export, there too. Unfortunately #4758 is still open.

Reply:
How is this solved for the `c` plugin? It's "write-only" too.

Also, the tests should really not try to import with a plugin that doesn't list `getstorage` in `infos/placements`.

But yes, in general I agree that `infos/status` is chaos and needs work...

Reply:
> How is this solved for the c plugin? It's "write-only" too.

Well the `c` plugin for one has info/status = `writeonly` (I've done this  here too now), and I've found a blacklist for write-only plugins in the shell tests. I added this plugin there too now.